crates/pbh-helpers/Cargo.toml

```diff
@@ -1,6 +1,30 @@
 [package]
 name = "pbh-helpers"
-version = "0.1.0"
-edition = "2021"
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+license.workspace = true
 
 [dependencies]
+world-chain-builder-pbh.workspace = true
```

Comment:
Let's consider making workspace members standalone without using relying on workspace items. This way they're easier to copy and paste